### PR TITLE
Fix review comments

### PR DESCRIPTION
**Pull request by vithu30**
## Purpose
This PR adds changes to fix review comments

## Documentation
N/A

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag [hunk 1]
```diff
@@ -2,10 +2,10 @@
     var APIUtil = org.wso2.carbon.apimgt.impl.utils.APIUtil;
     var apiPath = "";
     var goBack = "";
-    if(session.get('apiPath') != null){
+    if (session.get('apiPath') != null) {
         apiPath = session.get('apiPath');
     }
-    if(request.getParameter('goBack') != null){
+    if (request.getParameter('goBack') != null) {
```
Comment (mushthaq33) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag]: We can change this as in [1] as well :)
[1] - https://github.com/wso2/carbon-apimgt/pull/5392#discussion_r271594297

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag [hunk 2]
```diff
@@ -2,10 +2,10 @@
     var APIUtil = org.wso2.carbon.apimgt.impl.utils.APIUtil;
     var apiPath = "";
     var goBack = "";
-    if(session.get('apiPath') != null){
+    if (session.get('apiPath') != null) {
```
Comment (mushthaq33) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag]: We can change this as in [1] as well :)
[1] - https://github.com/wso2/carbon-apimgt/pull/5392#discussion_r271594297

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag (hunk 1 above)
Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag]: Resolved

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag (hunk 2 above)
Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag]: Resolved

**Pull request merged**